Clean list of Operations supported by fence and quiet (#6)
naveen-rn/specification: content/shmem_quiet.tex

\apisummary{

>     Waits for completion of all outstanding operations on symmetric data objects

Reviewer: ```suggestion
    Waits for completion of outstanding operations on symmetric data objects
```

Suggest removing "all", as the outstanding operations are limited in scope to a (implicit or explicit) context.

\end{apiarguments}

\apidescription{
    The \FUNC{shmem\_quiet} routine ensures completion of all operations
    on symmetric data objects issued by the calling \ac{PE} on the given context.
    Table~\ref{mem-order} lists the operations for which the

>     \FUNC{shmem\_quiet} routine ensures completion..

Reviewer 2: DocEdit: Remove the double period at the end of this line.

>     completion by \FUNC{shmem\_quiet} routine.

Reviewer: ```suggestion
    Table~\ref{mem-order} lists the operations for which the
    \FUNC{shmem\_quiet} routine ensures completion..
```

>     All operations to symmetric data objects are guaranteed to be completed and

Reviewer: ```suggestion
    All operations on symmetric data objects are guaranteed to be complete and
```

\apinotes{
    \FUNC{shmem\_quiet} is most useful as a way of ensuring completion of

>     several operations to symmetric data objects initiated by the calling

Reviewer: ```suggestion
    several operations on symmetric data objects initiated by the calling
```

    \ac{PE}. For example, one might use \FUNC{shmem\_quiet} to await delivery
    of a block of data before issuing another \PUT{} or nonblocking
    \PUT{} routine, which sets a completion flag on another \ac{PE}.
    \FUNC{shmem\_quiet} is not usually needed if
    \FUNC{shmem\_barrier\_all} or \FUNC{shmem\_barrier} are called.  The barrier

>     routines wait for the completion of outstanding writes to

Reviewer 2: "outstanding writes" might be more consistently phrased as **"outstanding operations"**.  "outstanding requests" would also be okay to me.

Author: used "outstanding operations" consistently

    symmetric data objects on all \acp{PE}.

    In an \openshmem program with multithreaded \acp{PE}, it is the
    user's responsibility to ensure ordering between operations issued by the
    threads in a \ac{PE} that target symmetric memory
    (e.g. \PUT{}, \ac{AMO}, \OPR{put-with-signal}, memory stores,

>     and nonblocking routines)

Reviewer 2: Similar to my note in `shmem_fence` above, is there a reason this `(e.g. ...)` wasn't omitted?
